can/interfaces/canalystii.py

```diff
@@ -0,0 +1,168 @@
+from ctypes import *
+import logging
+import platform
+from can import BusABC, Message
+
+logger = logging.getLogger(__name__)
+
+
+class VCI_INIT_CONFIG(Structure):
+    _fields_ = [("AccCode", c_int32),
+                ("AccMask", c_int32),
+                ("Reserved", c_int32),
+                ("Filter", c_ubyte),
+                ("Timing0", c_ubyte),
+                ("Timing1", c_ubyte),
+                ("Mode", c_ubyte)]
+
+
+class VCI_CAN_OBJ(Structure):
+    _fields_ = [("ID", c_uint),
+                ("TimeStamp", c_int),
+                ("TimeFlag", c_byte),
+                ("SendType", c_byte),
+                ("RemoteFlag", c_byte),
+                ("ExternFlag", c_byte),
+                ("DataLen", c_byte),
+                ("Data", c_ubyte * 8),
+                ("Reserved", c_byte * 3)]
+
+
+VCI_USBCAN2 = 4
+
+STATUS_OK = 0x01
+STATUS_ERR = 0x00
+
+try:
+    if platform.system() == "Windows":
+        CANalystII = WinDLL("./ControlCAN.dll")
+    else:
+        CANalystII = CDLL("./libcontrolcan.so")
+    logger.info("Loaded CANalystII library")
+except OSError as e:
+    CANalystII = None
+    logger.info("Cannot load CANalystII library")
+
+
+class CANalystIIBus(BusABC):
+    def __init__(self, channel, device=0, baud=10000000, Timing0=0x00, Timing1=0x14, can_filters=None):
+        """
+
+        :param channel: channel number
+        :param device: device number
+        :param baud: baud rate
+        :param Timing0: customize the timing register if baudrate is not specified
```

Comment:
Should these timing overrides default to `None`?

Reply:
yes, good catch